### PR TITLE
docs: Cleanup & prepare style guide for ES6

### PR DESCRIPTION
**Pull request by intih**
Some preparatory work for starting our ES6 style guide.

Renamed K&L > Education; moved JS style guide to ES5 style guide; added skeleton ES6 guide; removed `FUTURE.md` as it's not really appropriate/required in the style guide.

### CODE REVIEW

File: js.es5.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# K&L JavaScript Style Guide
+# Education ES5 JavaScript Style Guide
```
Comment (stuartmemo) [inline js.es5.md]: Not sure of the benefit of this? The style guide is just as relevant to ES6. I don't see any ES5-only things in there.

Comment (Jops) [inline js.es5.md]: We we're having a discussion yesterday about the style guide and the most efficient Es6 styles as part of a heated PR review (much wrath wow).
Things like `let` scope and where you put the variable definition came up. Won't break in an  Es5 style, but some prefer it updated for optimal Es6.

Comment (stuartmemo) [inline js.es5.md]: Yeah, so I would say those are features of the language rather than anything to do with style.

Comment (intih) [inline js.es5.md]: As discussed, will keep the 2 separate files (following AirBnB's example) - keeping the ES5 guide for reference for updating old codebases.

**Comment by intih** [comment]
(Approved by @stuartmemo)

**Pull request merged**